askup/views.py

```diff
@@ -1,3 +1,67 @@
-from django.shortcuts import render
+""" Views of the Askup application """
 
-# Create your views here.
+from django.views import generic
+
+from .models import Qset, Question
+
+
+class OrganizationsView(generic.ListView):
+    """ Handle the User Organizations list view """
+    template_name = 'askup/organizations.html'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['main_title'] = 'Your organizations'
+        return context
+
+    def get_queryset(self):
+        return Qset.objects.filter(parent_qset=None).order_by('name')
+
+
+class OrganizationView(generic.ListView):
+    """ Beholds the Organization's root qsets list view """
+    template_name = 'askup/organization.html'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['main_title'] = Qset.objects.get(id=self.kwargs.get('pk')).name
```

Comment:
I would handle get method in the try-except block. Otherwise, we could have 500 error here.